Makefile

```diff
@@ -0,0 +1,13 @@
+REPORT = full
+
+vendor: composer.json
+	composer install
+
+.PHONY: check
+check: vendor
```

Comment:
Waarom is de keuze hier gemaakt voor check i.p.v test?

Reply:
Om meer consistent te zijn met de Make [standard targets](https://www.gnu.org/software/make/manual/html_node/Standard-Targets.html).

Reply:
Ok. Maar dan vraag ik mij af:

Is er een verschil tussen deze tests en de tests die we normaal schrijven?
Of gebruiken we in de andere repositories een verkeerde term hiervoor en moet dat geupdate worden?

Reply:
Er is geen "verkeerde" term, maar aangezien we al veel van de standard targets gebruiken lijkt het mij een goed idee om dat voor tests ook te doen.

Reply:
Snap dat er geen verkeerde term is. Maar als dit inderdaad een soort standaard is en die ook willen gebruiken ben ik wel voorstander van om dit in elke repo gelijk te trekken zodat we niet overal verschillende termen gebruiken voor hetzelfde doel.